runtime/compiler/runtime/J9CodeCache.cpp

```diff
@@ -146,6 +146,19 @@ J9::CodeCache::initialize(TR::CodeCacheManager *manager,
    if (!self()->OMR::CodeCache::initialize(manager, codeCacheSegment, allocatedCodeCacheSizeInBytes))
       return false;
 
+
+   if (OMR::RSSReport::instance())
+      {
+      J9JavaVM * javaVM = jitConfig->javaVM;
+      PORT_ACCESS_FROM_JAVAVM(javaVM); // for j9vmem_supported_page_sizes
+
+      _coldCodeRSSRegion = new (PERSISTENT_NEW) OMR::RSSRegion("cold cache", _coldCodeAlloc, 0,
```

Comment:
I am guessing `_coldCodeRSSRegion` is defined in OMR. Is it initialized to NULL?

Reply:
Yes

Reply:
It may be worth asserting that `_coldCodeRSSRegion` is null to catch a potential future change in OMR that could allocate some (persistent) memory that then gets leaked by this code.

Reply:
Done